[IMP] IoT: edits to all IoT docs (#3673)
odoo/documentation: content/applications/productivity/iot/devices/scale.rst

===============
Connect a scale
===============

> When using the :abbr:`IoT (Internet of Things)` box in Odoo, there could be the need to use a scale.

Author: Noted, thanks for suggesting, I've changed this.

> Configuring one can be done in a few steps. After setup, the *Point of Sale app* can be used to

Reviewer: ```suggestion
Configuring one can be done in a few steps. After setup, the *Point of Sale* app can be used to
```

weigh products, which is helpful if their prices are calculated based on weight.

Connection
==========

To link the scale to the :abbr:`IoT (Internet of Things)` box, connect it with a :abbr:`USB
(Universal Serial Bus)` cable.

.. note::
   In some cases, a serial port to :abbr:`USB (Universal Serial Bus)` adapter may be needed.

If the scale is `compatible with Odoo IoT Box <https://www.odoo.com/page/iot-hardware>`_, there is
no need to set up anything because it will be automatically detected as soon as it is connected.

.. image:: scale/iot-choice.png
   :align: center
   :alt: IOT box auto detection.

The :abbr:`IoT (Internet of Things)` box may need to be restarted and the scale's drivers may need
to be downloaded to the box in some cases. To do so, go to the :guilabel:`IoT Box Home Page` and
click on :guilabel:`Drivers List`. Then, click on :guilabel:`Load Drivers`.

.. image:: scale/driver-list.png
   :align: center
   :alt: View of the IoT box settings and driver list.

> If loading the the drivers still doesn't allow for the scale to function it may be that the scale is
> not compatible with the Odoo :abbr:`IoT (Internet of Things)` box. In this case a different scale
> will need to be used.

Reviewer 2: Deleting an extra word and adding commas.
```suggestion
If loading the drivers still doesn't allow for the scale to function, it may be that the scale is
not compatible with the Odoo :abbr:`IoT (Internet of Things)` box. In this case, a different scale
will need to be used.
```

Use a scale in point of sale
============================

> To use the scale in the *Point of Sale app*, go to :menuselection:`PoS app --> 3-Dot Menu on the PoS

Reviewer: ```suggestion
To use the scale in the *Point of Sale* app, go to :menuselection:`PoS app --> 3-Dot Menu on the PoS
```

3-Dot Menu on the PoS --> Settings`, then enable the :abbr:`IoT (Internet of Things)` box feature.
After this is complete, the scale device can be set.

Select the scale from the :guilabel:`Electronic Scale` drop-down menu. Then click :guilabel:`Save`
to save the changes if required.

.. image:: scale/electronic-scale-feature.png
   :align: center
   :alt: List of the external tools that can be used with PoS and the IoT box.

> The scale is now available in all the :menuselection:`PoS` sessions. Now, if a product has a price

Reviewer 2: It might be more useful/appropriate to use an abbreviation tag here instead of a menuselection tag.
```suggestion
The scale is now available in all the :abbr:`PoS (Point of Sale)` sessions. Now, if a product has a price
```

Connect a Scale
===============

> When using the **IoT Box** in Odoo, there could be the need to use a scale. Configuring one is easy

Reviewer 3: ```suggestion
When using the *IoT Box* in Odoo, there could be the need to use a scale. Configuring one
```
This is *italicized* everywhere else so just keeping it consistent

> and convenient and it can be done in a few steps. After setup the :menuselection:`Point of Sale` app

Reviewer 3: ```suggestion
can be done in a few steps. After setup, the :menuselection:`Point of Sale` app
```

can be used to weigh products, which is helpful if their prices are calculated based on it.

Connection
==========

> To link the scale to the **IoT Box**, connect it with a USB cable.

Reviewer 3: ```suggestion
To link the scale to the :abbr:`IoT (Internet of Things)` Box, connect it with a USB cable.
```

.. note::
   In some cases, a serial port to USB adapter may be needed.

If the scale is `compatible with Odoo IoT Box <https://www.odoo.com/page/iot-hardware>`_, there
is no need to set up anything because it will be automatically detected as soon as it is connected.

.. image:: scale/iot-choice.png
   :align: center
   :alt: IOT box auto detection.

> The **IoT Box** may need to be restarted and the scales' drivers may need to be downloaded to the

Reviewer 3: ```suggestion
The :abbr:`IoT (Internet of Things)` Box may need to be restarted and the scales' drivers may need to be downloaded to the
```

Use a Scale in Point of Sale
============================

To use the scale in the :menuselection:`Point of Sale` app, go to :menuselection:`Point of Sale -->
Configuration --> Point of Sale`, open the :menuselection:`POS` that the scale will be configured
on, and enable the :guilabel:`IoT Box` feature (under :menuselection:`Connected Devices`).

.. image:: scale/iot-box-pos.png
   :align: center
   :alt: View of the IoT box feature inside of the PoS settings.

Now, choose the **IoT Box** in the dropdown menu and check the :guilabel:`Electronic Scale` option.

> Then, click on save.

Reviewer 3: ```suggestion
Then, click :guilabel:`Save`.
```

.. image:: scale/electronic-scale-feature.png
   :align: center
   :alt: List of the external tools that can be used with PoS and the IoT box.

> .. Needs new screenshot.

Reviewer 3: Still needs one or already added?

Author: It does still need a new screenshot, I'll get this added on.

> Use a Scale in Point of Sale

Reviewer 4: ```suggestion
Use a scale in point of sale
```

============================

To use the scale in the :menuselection:`Point of Sale` app, go to :menuselection:`Point of Sale -->
Configuration --> Point of Sale`, open the :menuselection:`POS` that the scale will be configured
on, and enable the :guilabel:`IoT Box` feature (under :menuselection:`Connected Devices`).

.. image:: scale/iot-box-pos.png
   :align: center
   :alt: View of the IoT box feature inside of the PoS settings.

Now, choose the :guilabel:`IoT Box` in the dropdown menu and check the :guilabel:`Electronic Scale`
option. Then, click :guilabel:`Save`.

.. image:: scale/electronic-scale-feature.png
   :align: center
   :alt: List of the external tools that can be used with PoS and the IoT box.

> The scale is now available in all the :menuselection:`PoS` sessions. Then, if a product has a price

Reviewer 4: ```suggestion
The scale is now available in all the :menuselection:`PoS` sessions. Now, if a product has a price
```
